cpp/json_schema_converter.cc

```diff
@@ -652,6 +654,122 @@ std::string JSONSchemaConverter::VisitAny(
          kBasicArray + " | " + kBasicObject;
 }
 
+std::string generateRangeRegex(std::optional<int> start, std::optional<int> end) {
```

Comment:
Could you make generateRangeRegex a static member function of JSONSchemaConverter, and follow the naming conversion (GenerateRangeRegex)?